Expensify/App
# Start a chat with anyone

### Details
<Explanation of the change or anything fishy that is going on>

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/145241

### Tests
1. Create an account. Add a firstname, lastname and profile photo.
2. Open expensify cash and login with your expensify email.
3. Search for the account in step 1. Click the account and confirm the profile photo, first and names show up,
![image](https://user-images.githubusercontent.com/14356145/103041898-9a079d80-452c-11eb-8deb-e53204ae46ce.png)
4. Search for an email that does not exist in your account (say test@test123456.com), click the email to DM it and confirm you can send a message to the account. If you check the db you'll see its a closed account. This means that the user will get an email of the chat inviting them to use expensify cash.
5. Search for a phone number (say 5857527440), confirm +1 gets automatically prefixed. Click on the phone to DM it and confirm you can send a message to the account. Once again if you check the db you'll see its a closed account.
6. Type the phone number again, confirm you can edit and place whatever international code you want. Say we put +415857527440. click the phone number and confirm you see this error because our backend couldn't validate it.
![image](https://user-images.githubusercontent.com/14356145/103042125-64af7f80-452d-11eb-90ac-8c85b496ff64.png)
7. Type the email test@expensify.cash and click it. Confirm you see the error,
![image](https://user-images.githubusercontent.com/14356145/103042167-90cb0080-452d-11eb-8561-b1bde6e97712.png)
